code/style.css

```diff
@@ -1,58 +1,264 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: 'Open Sans', sans-serif;
+  font-family: 'Prata', serif;
+}
+
+h4 {
+  color: #000000;
+  font-weight: 300;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 23px;
 }
 
 p {
-  margin: 1px;
+  margin: 10px;
   font-weight: 300;
-  background-color: #fff;
+  border-radius: 10px;
+  padding: 10px;
+  font-size: 20px;
+  color: #000000;
+  font-family: 'Open Sans', sans-serif;
+}
```

Comment:
If you find yourself adding the same properties for multiple selectors, one way could be to group them for several elements. In the above example, an alternative way to structure it could be:

```
h4, p {
  color: #000;
  font-family: 'Open Sans', sans-serif;
}

h4 {
  font-weight: 300;
  font-size: 23px;
}

p {
  margin: 10px;
  font-weight: 300;
  border-radius: 10px;
  padding: 10px;
  font-size: 20px;
}
```

Not a great example since it ended up using _more_ lines of code. But I find it's a zen-ish thing to do when I end up accidentally having many lines of duplicate properties scattered all over my CSS file.